CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Provide `--overwrite` flag to overwrite existing zimfiles
```

Comment:
Please add PR number in parenthesis (since we do not have an issue in warc2zim repo, otherwise we prefer to use issue number)